client/src/components/img-button/img-button.js

```diff
@@ -0,0 +1,62 @@
+import Component from '@/components/component';
+
+import ImgBox from '@/components/img-box/img-box';
+import DefaultImg from '@/assets/images/button-default.svg';
+
+import styles from '@/styles/components/img-button/img-button.module.scss';
+
+const MAX_IMAGE_UPLOAD = 10;
+
```

Comment:
```suggestion

// props : { buttonType, imageURL, onClickAdd, onClickRemove }
```

원래는 onClick과 onRemove 였는데 onClick은 괜찮은데 onRemove는 이 버튼 자체가 없어졌을때 이벤트 같아서 onClickRemove로 변경했으면 좋겠습니다.
이름을 변경하는 김에 좀더 의미 있게 onClickAdd로 바꾸면 좋겠습니다.

Reply:
네. 지금 보니 너무 모호한 이름이었네요. 수정했습니다.

Reply:
주석 넣어주세요~ 안넣어 주셨어요~